charts/folder-config-connector/templates/folder.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if and .Values.name (not .Values.folders) }}
+---
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: Folder
+metadata:
+  labels:
+  {{- include "folder-config-connector.labels" . | nindent 4 }}
+    {{- if .Values.labels }}
+    {{- toYaml .Values.labels | nindent 4 }}
+  {{- end  }}
+  {{- with .Values.annotations  }}
+  annotations:
+    {{- toYaml . | nindent 4 }}  
+  {{- end  }}
+  name: {{ .Values.name | quote }}
+spec:
+  {{- if .Values.displayName }}
+  displayName: {{ .Values.displayName | quote }}
+  {{- end }}
+  {{- if .Values.folderRef }}
+  folderRef:
+    {{- with .Values.folderRef }}
+    external: {{ .external | default nil | quote }}
+    name: {{ .name | default nil }}
+    namespace: {{ .namespace | default nil }}
+    {{- end }}
+    {{- end }}
+  {{- if .Values.organizationRef }}
+  organizationRef:
+  {{- with .Values.organizationRef }}
+    external: {{ .external | default nil | quote }}
+    name: {{ .name | default nil }}
+    namespace: {{ .namespace | default nil }}
+  {{- end }}
+  {{- end }}
+  {{- if .Values.resourceID }}
+  resourceID: {{ .Values.resourceID | toString | quote }}
```

Comment:
why toString function? and quote function? I think it is a double protection against the same.

Reply:
now usage of quote is consistent